packages/progress-bar/src/ProgressBar.stories.tsx

```diff
@@ -165,73 +143,70 @@ export const WithHeaderTruncation: StoryObj<typeof ProgressBar> = {
   },
 };
 
-export const DeterminateVariants: StoryObj<typeof ProgressBar> = {
+export const IndeterminateVariants: StoryObj<typeof ProgressBar> = {
   parameters: {
     generate: {
       combineArgs: {
-        variant: LOADER_VARIANTS,
+        variant: ANIMATED_VARIANTS,
+      },
+      args: {
+        ...requiredA11yArgs,
+        isIndeterminate: true,
+        value: storyValues.value,
+        formatValue: (value: number) => `${value} MBs`,
+        showIcon: true,
+      },
+    },
+  },
+};
+
+export const DeterminateProgressVariants: StoryObj<typeof ProgressBar> = {
+  parameters: {
+    generate: {
+      combineArgs: {
+        variant: VARIANTS,
         enableAnimation: [false, true],
         disabled: [false, true],
       },
       args: {
-        ...defaultArgs,
-        isIndeterminate: false,
+        ...sharedDeterminateArgs,
         formatValue: (value: number, maxValue?: number) =>
           `${value} / ${maxValue} GB`,
         showIcon: true,
       },
       excludeCombinations: [
         {
-          variant: LOADER_VARIANTS.filter(
-            v =>
-              !(ANIMATED_LOADER_VARIANTS as Array<LoaderVariant>).includes(v),
+          variant: VARIANTS.filter(
+            v => !(ANIMATED_VARIANTS as Array<Variant>).includes(v),
           ),
           enableAnimation: [true],
```

Comment:
I also expect these animated tests might be flaky in chromatic (the snapshot will be taken at a slightly different point in the animation each time)